pandas/io/sql.py

```diff
@@ -644,6 +644,37 @@ def read_sql(
     read_sql_table : Read SQL database table into a DataFrame.
     read_sql_query : Read SQL query into a DataFrame.
 
+    Notes
+    -----
+    Using string interpolation (e.g. ``f-strings``, ``%-formatting``,
+    ``str.format()``, etc.) in a SQL query may cause SQL injection.
```

Comment:
I think I might re-word this to "may allow SQL injection if query inputs aren't sanitized"

Reply:
Thanks! I re-worded this line for better understanding